src/activities/Elsa.Activities.Http/Activities/SendHttpRequest/SendHttpRequest.cs

```diff
@@ -231,14 +240,20 @@ private HttpRequestMessage CreateRequest()
         }
         
         object? IActivityPropertyOptionsProvider.GetOptions(PropertyInfo property)
-        {
-            if (property.Name != nameof(ResponseContentParserName))
-                return null;
-            
-            var items =  _parsers.Select(x => new SelectListItem(x.Name, x.Name)).ToList();
-            
-            items.Insert(0, new SelectListItem("Auto Select", ""));
-            return items;
+        { 
+            switch (property.Name)
+            {
+                case nameof(ResponseContentParserName):
+                {
+
+                    var items = _parsers.Select(x => new SelectListItem(x.Name, x.Name)).ToList();
+
+                    items.Insert(0, new SelectListItem("Auto Select", ""));
+                    return items;
+                }
+                case nameof(ResponseContentTargetType): return null;
+                default: throw new ArgumentException($"Unsupported property: {property.Name}");
```

Comment:
I don't think it's a good idea to throw an exception here, because if we add a new property to the activity, we have to add a case here for no other reason as not to throw.

Since this method only applies to the `ResponseContentParserName` property, I suggest to revert this change to its original form (unless you added something useful that I missed?)